addons/google_calendar/models/res_users.py

```diff
@@ -114,3 +117,19 @@ def _sync_all_google_calendar(self):
                 user.with_user(user).sudo()._sync_google_calendar(google)
             except Exception as e:
                 _logger.exception("[%s] Calendar Synchro - Exception : %s !", user, exception_to_unicode(e))
+
```

Comment:
(sorry, I can't highlight the relevant line :shrug: )

Line #112: you should add `google_synchronization_stopped != False` in the domain. That way the cron will not attempt to start the synchronization for users who disabled it.